src/mon/LogMonitor.cc

```diff
@@ -563,12 +591,7 @@ void LogMonitor::check_sub(Subscription *s)
 
   if (s->next == 0) { 
     /* First timer, heh? */
-    bool ret = _create_sub_summary(mlog, sub_level);
-    if (!ret) {
-      dout(1) << __func__ << " ret = " << ret << dendl;
-      mlog->put();
-      return;
-    }
+    _create_sub_incremental(mlog, sub_level, get_last_committed());
```

Comment:
Is this going to work if the latest message `sub_level` is in an older revision than `get_last_committed`?

Reply:
Never mind, the old _create_sub_summary was only looking at the current summary anyway